README.md

```diff
@@ -31,6 +39,131 @@ Create a `.env.docker` file to store your own custom env vars. See [`example.doc
 3. `docker ps` should show 2 docker containers (`gotrue_postgresql` and `gotrue_gotrue`)
 4. That's it! Visit the [health checkendpoint](http://localhost:9999/health) to confirm that gotrue is running.
 
+## Running in production
+
+Running an authentication server in production is not an easy feat. We
+recommend using [Supabase Auth](https://supabase.com/auth) which gets regular
+security updates.
+
+Otherwise, please make sure you setup a process to promptly update to the
+latest version. You can do that by following this repository, specifically the
+[Releases](https://github.com/supabase/gotrue/releases) and [Security
+Advisories](https://github.com/supabase/gotrue/security/advisories) sections.
+
+You should also subscribe to the
+[supabase-gotrue-announcements](https://groups.google.com/g/supabase-gotrue-announcements)
+Google Group to get email updates for important releases.
+
+### Backward compatibility
+
+GoTrue uses the [Semantic Versioning](https://semver.org) scheme. Here are some
+further clarifications on backward compatibility guarantees:
+
+**Go API compatibility**
+
+GoTrue is not meant to be used as a Go library. There are no guarantees on
+backward API compatibility when used this way regardless which version number
+changes.
+
+**Patch**
+
+Changes to the patch version guarantees backward compatibility with:
+
+- Database objects (tables, columns, indexes, functions).
+- REST API
+- JWT structure
+- Configuration
```

Comment:
Should we specify that this refers to the names of the configuration values which live in the [Environment Variable files](https://github.com/supabase/gotrue/blob/master/example.env)?

Imagine that we might want to change the properties of the logger down the road which are also part of the configuration atm